ui/src/main/res/values/styles.xml

```diff
@@ -149,4 +149,11 @@
     <style name="Style.Button.Flat" parent="Widget.AppCompat.Button.Borderless.Colored">
         <item name="android:textAppearance">?attr/textAppearance.Button</item>
     </style>
+
+    <style name="Style.ProgressBar.Horizontal" parent="Base.Widget.AppCompat.ProgressBar.Horizontal"/>
+
+    <style name="Style.Toolbar.TransparentTitle" parent="ThemeOverlay.AppCompat.Dark.ActionBar">
```

Comment:
I use it to disable title when toolbar is expanded. I could delete parent reference.